webdriver/tests/bidi/browsing_context/command_get_tree.py

```diff
@@ -0,0 +1,16 @@
+import pytest
```

Comment:
done. Split tests by domain + command/event. Serialization went to it's own domain.

Reply:
FYI: I'm going to do the refactoring of the folder organization for existing tests over on https://bugzilla.mozilla.org/show_bug.cgi?id=1752805 which should land later today.

Reply:
Thanks for the heads-up! I'm going to back to the PR later, maybe next week. So please go ahead and restructure the test if you need it.

Reply:
Will do. I won't necessarily need this particular test but `context_created.py`.

Reply:
I'm working on this command next. So you can also expect some more tests to be added soon.

Reply:
@sadym-chromium FYI a complete set of wdspec tests will be merged soon via https://github.com/web-platform-tests/wpt/pull/33258. As such it might be good to rebase your branch against latest tip. Thanks.

Reply:
thanks for the update! Unfortunately I still don't work on the PR.

Reply:
No problem at all. It was really just a FYI. Also note that @juliandescottes started to work on `browsingContext.navigate`. So maybe we wont need this PR anymore?